Digital Credentials: exclude various swift files

marcoscaceres opened the pull request:
#### b245f4b498559bf646454baf91c65fa3110dc873
<pre>
Digital Credentials: exclude various swift files
<a href="https://bugs.webkit.org/show_bug.cgi?id=292723">https://bugs.webkit.org/show_bug.cgi?id=292723</a>
<a href="https://rdar.apple.com/150934476">rdar://150934476</a>

Reviewed by Aditya Keerthi.

Exclude various swift stubs required by the Digital Credentials API.

Canonical link: <a href="https://commits.webkit.org/294814@main">https://commits.webkit.org/294814@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/de5a3a695b87383c2d5980d1ea6f004fec9d6179

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/102996 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/22671 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/12990 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/108164 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/53637 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/23015 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/31171 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/78296 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/35246 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/106002 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/17830 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/92932 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/58631 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/17669 "Passed tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/52994 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/87479 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/11042 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/110537 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/30133 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/22200 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/87284 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/30498 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/89135 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/86908 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/22152 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/31757 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/9474 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/24411 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/30060 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/35382 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/29868 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/33195 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/31430 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aprotyas commented on Source/WebKit/Configurations/WebKit.xcconfig:
```diff
@@ -316,6 +316,18 @@ WK_EXCLUDED_MATERIAL_FILES_ = MaterialAdditions.swift;
 WK_EXCLUDED_PDF_INDICATOR_FILES = $(WK_EXCLUDED_PDF_INDICATOR_FILES_$(USE_INTERNAL_SDK));
 WK_EXCLUDED_PDF_INDICATOR_FILES_ = WKPDFPageNumberIndicatorAdditions.swift;
 
+_EXCLUDED_IDENTITY_SERVICES_FILES = $(WK_EXCLUDED_IDENTITY_SERVICES_FILES_$(USE_INTERNAL_SDK));
```
Shouldn't this be `WK_EXCLUDED_IDENTITY_SERVICES_FILES` instead?
```suggestion
WK_EXCLUDED_IDENTITY_SERVICES_FILES = $(WK_EXCLUDED_IDENTITY_SERVICES_FILES_$(USE_INTERNAL_SDK));
```

marcoscaceres committed:
Digital Credentials: exclude various swift files
https://bugs.webkit.org/show_bug.cgi?id=292723
rdar://150934476

Reviewed by Aditya Keerthi.

Exclude various swift stubs required by the Digital Credentials API.

Canonical link: https://commits.webkit.org/294814@main
